APIs/FlowCompatibilityManagementAPI.raml

```diff
@@ -89,19 +89,24 @@ mediaType: application/json
               body:
                 type: !include schemas/error.json
         put:
-          description: Replace Active Constraints with other ones.
+          description: >
+            Validates proposed Constraints against JSON Schema of request body
+            and Supported Parameter Constraints.
           body:
             example: !include ../examples/constraints-active-get-200.json
             type: !include schemas/constraints_active.json
           responses:
             200:
-              description: Returned when success.
+              description: >
+                Returned when this Sender is capable to adhere at least one
```

Comment:
This seems to be misleading ... the Sender accept the constraints if they are valid and included in the supported constraints ... and possibly if the extents of the constraints are supported by the Sender ... BUT it does not mean the Sender will be able to adhere to the constraints ... it all depends on the effective input signal.

Reply:
Maybe we need better wording here. My understanding is that ability/capability to adhere means the Sender knows these values and will try and doesn't guarantee that Sender will adhere after the PUT.